dev-tex/pythontex/pythontex-0.15.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 python3_{3,4} )
```

Comment:
python3_3 is off the tree you should put 3_4, 3_5 and possibly 3_6 if you know it is supported.